cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -251,6 +252,11 @@ def create_xblock(parent_locator, user, category, display_name, boilerplate=None
         # if we add one then we need to also add it to the policy information (i.e. metadata)
         # we should remove this once we can break this reference from the course to static tabs
         if category == 'static_tab':
+
+            tags = [ "action:{}".format(unicode(dest_usage_key.course_key)) ]
+            tags.append("location:create_xblock_static_tab")
```

Comment:
You don't need to use `append` here. You could just put both values in the list right off the bat.